docs/release-notes/2.6.0.asciidoc

```diff
@@ -31,6 +31,8 @@
 
 * Fix potential panic in Elasticsearch client equal function. {pull}6128[#6128]
 * Increment ECK-stack Helm chart version to support addition of Agent/Fleet Server. {pull}6179[#6179]
+* Fix version gate for the file-based settings volume {pull}6289[#6289]
```

Comment:
```suggestion
```
Not sure it's relevant to list it because it's part of * New CRD StackConfigPolicy to declaratively configure multiple Elasticsearch clusters.

Reply:
That's a good point. Removed